platform_plugin_forum_email_notifier/tasks.py

```diff
@@ -0,0 +1,57 @@
+"""Tasks for the platform_plugin_forum_email_notifier plugin."""
+import logging
+
+from celery import shared_task
+from django.contrib.auth import get_user_model
+from edx_ace.recipient import Recipient
+from edx_django_utils.monitoring import set_code_owner_attribute
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
```

Comment:
There's a method called: get_course_overview_or_none in edx-platform https://github.com/openedx/edx-platform/blob/fcc80eca30ab81ec9387eb264bcf4035d175025b/openedx/core/djangoapps/content/course_overviews/api.py#L17-L32